apps/condo/domains/organization/utils/accessSchema.js

```diff
@@ -41,6 +41,12 @@ async function checkRelatedOrganizationPermission (context, userId, organization
     return checkOrganizationPermission(context, userId, organizationLink.from.id, permission)
 }
 
+// TODO(nomerdvadcatpyat): use this function where checkRelatedOrganizationPermission and checkOrganizationPermission used together
+async function checkPermissionInUserOrganizationOrRelatedOrganization (context, userId, organizationId, permission) {
+    const canManageRelatedOrganizationMeters = await checkRelatedOrganizationPermission(context, userId, organizationId, permission)
+    const canManageUserOrganizationMeters = await checkOrganizationPermission(context, userId, organizationId, permission)
```

Comment:
You named constants as `…Meters`, but the function accepts generic permission name string.